src/richie/apps/core/management/commands/create_demo_site.py

```diff
@@ -111,28 +130,59 @@ def clear_cms_data():
     Person.objects.all().delete()
 
 
-def create_demo_site():
+def recursive_page_creation(site, pages, parent=None):
```

Comment:
I think it is becoming too complicated for just one child page, and you have created empty pages so the final solution where you add content to each page will not look like this... 

For the moment, maybe just add this to the create_demo_site method:
```
languages=[l[0] for l in settings.LANGUAGES]
about_page = create_i18n_page(
    languages=languages,
    parent=pages_created["annex"],
    published=True,
)
create_text_plugin(
    about_page,
    "main_content",
    languages=languages,
)